friendly/src/main/java/com/google/sps/servlets/GroupServlet.java

```diff
@@ -84,10 +84,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     for (String userEmail: members) {
         if (!validateGroupUser(userEmail)) {
-            response.getWriter().println("Invalid"); 
+            response.getWriter().println("User does not exist."); 
```

Comment:
Ah okay I see, thanks for improving the coding style here

Reply:
welcome!